lib/sigstore/signer.rb

```diff
@@ -109,35 +122,59 @@ def fetch_cert(csr)
 
       resp_body = JSON.parse(resp.body)
 
-      chain = resp_body.fetch("signedCertificateEmbeddedSct").fetch("chain")
-                       .fetch("certificates").map { |pem| Internal::X509::Certificate.read(pem) }
+      cert = resp_body.fetch("signedCertificateEmbeddedSct").fetch("chain")
```

Comment:
nit: validate there are `signedCertificateEmbeddedSct` in the response, we don't want to handle detached SCTs

Reply:
You can assume you will not received detached SCTs, we plan to mark them as deprecated - https://github.com/sigstore/fulcio/issues/1499